provider/lxd/environ.go

```diff
@@ -231,7 +236,8 @@ func (env *environ) Destroy(ctx context.ProviderCallContext) error {
 			return errors.Annotate(err, "destroying LXD filesystems for model")
 		}
 	}
-	return nil
+
+	return env.DestroyProfiles()
```

Comment:
Could we get the error and annotate , eg above we annotate with  "destroying LXD filesystems for model"